docs/changelog.md

```
@@ -1,5 +1,9 @@
 # Changelog
 
+### Unreleased
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)